pkg/quota/admission/clusterresourceoverride/admission.go

```diff
@@ -251,7 +260,57 @@ func updateContainerResources(config *internalConfig, container *kapi.Container)
 		if cpuFloor.Cmp(*q) > 0 {
 			q = cpuFloor.Copy()
 		}
+		if nsCPUFloor != nil && q.Cmp(*nsCPUFloor) < 0 {
+			glog.V(5).Infof("%s: %s pod limit %q below namespace limit; setting limit to %q", api.PluginName, kapi.ResourceCPU, q.String(), nsCPUFloor.String())
+			q = nsCPUFloor.Copy()
+		}
 		resources.Requests[kapi.ResourceCPU] = *q
 	}
+}
+
+// minResourceLimits finds the minimum CPU and minimum Memory resource
+// values across all the limits in limitRanges. Nil is returned if
+// there is CPU or Memory resource respectively.
+func minResourceLimits(limitRanges []*kapi.LimitRange) (*resource.Quantity, *resource.Quantity) {
```

Comment:
nit: I think this is clearer if minResourceLimits(limits, resourceName) quantity

Then calling side you can just call twice with cpu and memory